From c610f841ac50a2dbd7417f8f1bcc5d27260b62c9 Mon Sep 17 00:00:00 2001
Message-Id: <c610f841ac50a2dbd7417f8f1bcc5d27260b62c9.1464085843.git.pmatilai@redhat.com>
From: Panu Matilainen <pmatilai@redhat.com>
Date: Tue, 24 May 2016 13:20:47 +0300
Subject: [PATCH] qede: add missing external dependency and disable by default

The qede driver depends on libz but the LDLIBS entry in makefile
was missing. Also because of the external dependency, make it
disabled in default config as per common DPDK policy on external deps.

Fixes: ec94dbc57362 ("qede: add base driver")

Signed-off-by: Panu Matilainen <pmatilai@redhat.com>
---
 config/common_base        | 2 +-
 drivers/net/qede/Makefile | 2 ++
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/config/common_base b/config/common_base
index 3535c6e..47c26f6 100644
--- a/config/common_base
+++ b/config/common_base
@@ -299,7 +299,7 @@ CONFIG_RTE_LIBRTE_BOND_DEBUG_ALB_L1=n
 
 # QLogic 25G/40G PMD
 #
-CONFIG_RTE_LIBRTE_QEDE_PMD=y
+CONFIG_RTE_LIBRTE_QEDE_PMD=n
 CONFIG_RTE_LIBRTE_QEDE_DEBUG_INIT=n
 CONFIG_RTE_LIBRTE_QEDE_DEBUG_INFO=n
 CONFIG_RTE_LIBRTE_QEDE_DEBUG_DRV=n
diff --git a/drivers/net/qede/Makefile b/drivers/net/qede/Makefile
index 4cc9ee8..c9b3b1c 100644
--- a/drivers/net/qede/Makefile
+++ b/drivers/net/qede/Makefile
@@ -14,6 +14,8 @@ LIB = librte_pmd_qede.a
 CFLAGS += -O3
 CFLAGS += $(WERROR_FLAGS)
 
+LDLIBS += -lz
+
 EXPORT_MAP := rte_pmd_qede_version.map
 
 LIBABIVER := 1
-- 
2.5.5

